src/components/BusinessList/BusinessList.js

```diff
@@ -1,40 +1,37 @@
-import React, { useState, useEffect } from 'react';
-// import Axios from 'axios';
+import React from 'react';
+import Axios from 'axios';
 import Spinner from '../LoadSpinner/LoadSpinner';
 import BusinessCard from '../BusinessCard/BusinessCard';
+import { Scraper } from '../../utils';
 
-const businessList = () => {
-  const [businesses, setBusinesses] = useState([]);
-
-  useEffect(() => {
-    // TODO: implement something like this eventually
-    // Axios.get(
-    //   'https://redirect-blm.herokuapp.com/api/business/getBusinesses'
-    // ).then(res => {
-    //   setBusinesses(res.data);
-    // });
-    var temp = [
-      {
-        title: 'Hope for Flowers',
-        description: 'sustainable clothing for women',
-        website: 'https://hopeforflowers.com/'
-      },
-      {
-        title: 'Fubu',
-        description: 'hip hop apparel',
-        website: 'https://fubu.com/'
-      },
-      {
-        title: 'Hanifa',
-        description: 'apparel for women',
-        website: 'https://hanifa.co/'
+class BusinessList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      lst: [],
+      error: null
+    }
+  }
+  componentDidMount() {
+    const component = this;
+    const category = Scraper.category(window.document);
+    Axios({
+      method: 'get',
+      withCredentials: true,
+      baseUrl: 'https://redirect-blm.herokuapp.com/api/business/getone/category',
```

Comment:
Is category in this URL meant to come from the category variable in line 17? Should this be string interpolation, as follows?

``https://redirect-blm.herokuapp.com/api/business/getone/${category}``

Reply:
Yea you would use string interpretation in the axios.get method e.g. `axios.get(<url>)`

Since im using the axios constructor here it's not necessary